ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -58,9 +58,6 @@ class LargeTitleView: UIView {
         }
     }
 
-    var avatarHeightConstraint: NSLayoutConstraint?
```

Comment:
what was the original change/commit/pr? what was the bug?

Reply:
and actually mention it in your pr and commit message.

Reply:
https://github.com/microsoft/fluentui-apple/pull/919#discussion_r810401567
There wasn't really a good way to get the size to the LargeTitleView, so I checked if it worked without the constraints, and it did.